src/SourceBuild/content/eng/Versions.props

```diff
@@ -15,23 +15,11 @@
       Building .NET from source depends on several archives, depending on the branch's current
       source-buildability status.
 
-      PrivateSourceBuiltArtifactsUrl is a tar.gz of .NET build outputs from a previous
-      build needed to build the current version of .NET. This is always defined, because .NET needs
-      to be bootstrappable at any point in time.
-
-      PrivateSourceBuiltPrebuiltsUrl is a tar.gz of assets downloaded from the internet
-      that are needed to build the current version of .NET. Early in the lifecycle of a .NET major
-      or minor release, prebuilts may be needed. When the release is mature, prebuilts are not
-      necessary, and this property is removed from the file.
-
-      PrivateSourceBuiltSdkUrl_<abc> is a tar.gz of the source build .NET SDK from the previous release
-      for a particular OS. It is used by CI to validate that the current release can be built with it.
-
-      These URLs can't be composed from their base URL and version as we read them from the
-      prep.sh and pipeline scripts, outside of MSBuild.
+      SourceBuiltVersionNumber is the version of a tar.gz that can be used to get 
+      a tar.gz of .NET build outputs from a previous build needed to build the current version of .NET,
+      a tar.gz of assets downloaded from the internet that are needed to build the current version of .NET, or
+      a tar.gz of the source build .NET SDK from the previous release for a particular OS.
     -->
-    <PrivateSourceBuiltArtifactsUrl>https://dotnetcli.azureedge.net/source-built-artifacts/assets/Private.SourceBuilt.Artifacts.8.0.100-rc.1.23455.1.centos.8-x64.tar.gz</PrivateSourceBuiltArtifactsUrl>
-    <PrivateSourceBuiltSdkUrl_CentOS8Stream>https://dotnetcli.azureedge.net/source-built-artifacts/sdks/dotnet-sdk-8.0.100-rc.1.23455.1-centos.8-x64.tar.gz</PrivateSourceBuiltSdkUrl_CentOS8Stream>
-    <PrivateSourceBuiltPrebuiltsUrl>https://dotnetcli.azureedge.net/source-built-artifacts/assets/Private.SourceBuilt.Prebuilts.0.1.0-9.0.100-3.centos.8-x64.tar.gz</PrivateSourceBuiltPrebuiltsUrl>
```

Comment:
There needs to be a control that explicitly indicates whether or not to pull the prebuilts tarball.  This is an excape hatch of sorts and is not the normal flow.

Reply:
I'm open to suggestions on how to do this. Currently, the prebuilts are pulled if the parameter <PrivateSourceBuiltPrebuiltsUrl> is specified. I've now changed it so that even if the parameter is specified, the prebuilts won't be downloaded unless the http status code for the URL is in the range of 200-399.